README.md

```diff
@@ -40,9 +40,23 @@ https://github.com/protocolbuffers/protobuf/releases
 Then, `go get -u` as usual the following packages:
 
 ```sh
+go get -u google.golang.org/grpc
```

Comment:
This shouldn't be necessary to use the generators, I think? Why did you add it?

Reply:
I am not sure because I have already installed grpc cpp++ in my labtop and workstation from source before I use grpc-gateway. I think we at least need to install grpc c-core hence I add it.

Reply:
No, you dont need it. Please remove it again. You need it to use the generated files, but you don't need it to generate it.